### DISCUSSION TITLE
fonts on forge provisioned server

### DISCUSSION THREAD

**Discussion by azc666** [spatie/laravel-pdf#122] [Q&A]
on a laravel 11 herd provisioned app I can create a pdf that honors app.css and tailwind.config.js. When I deploy to forge all the tailwind classes are recognized including custom colors, etc. but the fonts default to times. My question is: should I expect all my custom fonts to work as they do locally? I have installed puppeteer per the instructions for a forge provisioned server, but if the custom fonts should work I will continue to dig into that.

**Comment by talelmishali** [comment]
Hey @azc666, have you found any solution for that? I came across the same issue. Thanks 🙏🏻

**Reply by azc666** [reply]
My solution was to add the fonts directly on the server and reference that path as well as the local path. I don't remember the exact path but it was something like home/local/share or /usr/local/share. After u ssh in you can search for share to see other fonts that are already in that folder. Hope that helps.

**Reply by talelmishali** [reply]
Thanks! Will give it a try.
